src/main/java/org/myrobotlab/service/Tracking.java

```diff
@@ -303,6 +309,67 @@ public ServiceConfig getConfig() {
     return config;
   }
 
+  static public Map<String, Map<String, ServiceConfig>> getDefaultConfigs() {
+
+    Map<String, ServiceConfig> config = new HashMap<>();
+
+    RuntimeConfig runtime = new RuntimeConfig();
+    runtime.registry = new String[] { "controller", "cv", "tilt", "pan", "pid", "tracking" };

```

Comment:
this shouldn't be defined here.  we should change the return signature so that defining the runtime registery is unnecessary here.  see comment on the base class.